src/pattern-library/components/patterns/UsingComponentsPage.js

```diff
@@ -21,14 +21,23 @@ export default function UsingComponentsPage() {
     >
       <Library.Pattern title="Component categories">
         <ul>
-          <li>
-            <strong>Simple components</strong>: Opinionated, simple components
-            that apply design patterns. Example: <code>Spinner</code>.
-          </li>
           <li>
             <strong>Presentational components</strong>: Composable components
             with customization flexibility. Example: <code>Button</code>,{' '}
-            <code>Link</code>.
+            <code>Link</code>. This is the most common category of component.
+            <ul>
+              <li>
+                <strong>Base presentational components</strong>: A type of
+                presentational component that only applies minimal, core styles
+                for the {"component's"} primary element. This adds flexibility
```

Comment:
It would be useful to clarify what is "in scope" for "minimal, core styles". This might vary per component, but from the current examples it looks like there are some commonalities (eg. flex layout, whitespace handling, focus ring).